test/builders/build/eth-api/libraries/viem/send-transaction.js

```diff
@@ -0,0 +1,66 @@
+import { assert } from 'chai';
+import { createPublicClient, createWalletClient, http, parseEther } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import { moonbaseAlpha } from 'viem/chains';
+
+const fromAddressPrivateKey = "0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133"
+const fromAddress = "0xf24FF3a9CF04c71Dbc94D0b566f7A27B94566cac"
+const toAddress = "0x73328873A6Ac3DFc9455986358F5230cB3B8e92c"
+const url = process.env.HTTP_RPC_ENDPOINT
+
+describe('Viem - Send a Transaction', function () {
+    const account = privateKeyToAccount(fromAddressPrivateKey);
+    const rpcUrl = url;
+
+    // 2. Create a wallet client for writing chain data
+    const walletClient = createWalletClient({
+        account,
+        chain: moonbaseAlpha,
+        transport: http(rpcUrl),
+    });
+
+    // 3. Create a public client for reading chain data
+    const publicClient = createPublicClient({
+        chain: moonbaseAlpha,
+        transport: http(rpcUrl),
+    });
+
+    // 4. Create to address variable
+    const addressFrom = fromAddress;
+    const addressTo = toAddress;
+
+    var balanceFrom
+    var balanceTo
+
+    describe('Check Balances -  balances.js', async () => {
+        it('should return a balance for alice', async () => {
+            balanceFrom = await publicClient.getBalance({ address: addressFrom })
+            assert.equal(balanceFrom > 0, true);
+        });
+        it('should return a balance for bob', async () => {
+            balanceTo = await publicClient.getBalance({ address: addressTo })
+            assert.equal(balanceTo > 0, true);
```

Comment:
The balance of the randomly generated account should be `0` to begin with. So, this assertion will need to change